pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -103,7 +112,7 @@ func (o *ClusterInfoOptions) Run() error {
 		if err != nil {
 			return err
 		}
-		printService(o.Out, "Kubernetes master", o.Client.Host)
+		printService(o.Out, "Kubernetes master", o.Client.Host, !o.NoColor)
```

Comment:
`!o.NoColor` is equal to `o.Color`

That's why I'd prefer to use `o.Color` directly.